monitor/flowgen/types/types.go

```diff
@@ -16,12 +16,14 @@ type Job struct {
 	// Cadence is intrerval at which to run the job
 	Cadence time.Duration
 
-	Network netconf.ID
+	NetworkID netconf.ID
 
 	SrcChain uint64
 	DstChain uint64
 
 	Owner common.Address
 
+	InboxAddr common.Address
```

Comment:
this job struct is getting very cluttered with all kinds of data, many of which are constant for the network, so never change. The Job struct should only contain data unique to the job. But I can also take a go at cleaning this up later, no worries.